tests/drivers/build_all/gpio/app.overlay

```diff
@@ -210,7 +210,7 @@
 			clock-frequency = <2000000>;
 
 			/* one entry for every devices at spi.dtsi */
-			cs-gpios = <&test_gpio 0 0 &test_gpio 0 0 &test_gpio 0 0>;
+			cs-gpios = <&test_gpio 0 0 &test_gpio 0 0 &test_gpio 0 0 &test_gpio 0 0 &test_gpio 0 0>;
```

Comment:
probably worth stacking these up at this point? easier to copy paste more

```
cs-gpios = <&test_gpio 0 0>,
           <&test_gpio 0 0>, 
           ...
           <&test_gpio 0 0>;
```

Reply:
Fixed.